src/components/Proposal/SentimentSurvey/SentimentSurveyRow.css

```diff
@@ -0,0 +1,75 @@
+.SentimentSurveyRow {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-top: 1px solid var(--black-100);
+  min-height: 51px;
+  padding-right: 20px;
+  padding-left: 20px;
+  overflow: hidden;
+  position: relative;
+}
+
+.SentimentSurveyRow:hover {
+  background: var(--white-900);
+  border: 1px solid var(--black-100);
+  box-shadow: 0 0 15px var(--black-400);
+  border-radius: 6px;
+}
+
+.SentimentSurveyRow__AddReaction {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.SentimentSurveyRow__AddReaction > span {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 18px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--black-600);
+  position: relative;
+  opacity: 0;
+}
+
+.SentimentSurveyRow:hover .SentimentSurveyRow__AddReactionLabel {
+  opacity: 1 !important;
+}
+
+#slide {
```

Comment:
Maybe I'm missing something but can't you use `SentimentSurveyRow__AddReaction`?